src/screens/PutawayItem/index.tsx

```diff
@@ -37,6 +34,12 @@ class PutawayItem extends Component<Props, State> {
     const { selectedLocation, createPutawayOderAction } = this.props;
     const { item } = this.props.route.params;
     const { currentLocation } = this.props;
+
+    if (!this.state.selectedLocation?.length) {
```

Comment:
This is the exact opposite of what had to be done. We should allow empty selectedLocation which means `Default` bin.

Reply:
@hellochirag additionally it should be done on the pending putaway item, not on the putaway candidate

Reply:
@awalkowiak the error getting in putaway item because of the empty Putawat Location from Putway Candidate. during creating the Putaway item the location is null that's why the error throwing in Putaway item details

Reply:
Which Location do we need to allow in the default location @awalkowiak ?

Reply:
@hellochirag in both putaway candidate and pending putaway items we should allow setting `putawayLocation` into null. The candidate view was working as expected.

Reply:
Ok @awalkowiak I will revert my changes and remove the bin location empty alert